quiz.go

```diff
@@ -7,9 +7,11 @@ import (
 	"io"
 	"log"
 	"os"
+	"time"
 )
 
 func main() {
+	finished := make(chan bool)
```

Comment:
The standard practice for a channel that is used as a signal with the actual data being ignored and discarded is to use a `chan struct{}`. This has no memory overhead vs a bool which requires a tiny amount of memory.

```go
x := make(chan struct{})
go func() {x<-struct{}{}}()
<-x
```